mlflow/store/db/utils.py

```diff
@@ -100,6 +100,9 @@ def _get_alembic_config(db_url, alembic_dir=None):
     from alembic.config import Config
     final_alembic_dir = os.path.join(_get_package_dir(), 'store', 'db_migrations')\
         if alembic_dir is None else alembic_dir
+    # Escape any '%' that appears in a db_url. This could be in a password,
+    # url, or anything that is part of a potentially complex database url
+    db_url = db_url.replace('%', '%%')
```

Comment:
This change seems reasonable looking at the Alembic docs for `set_main_option` (https://alembic.sqlalchemy.org/en/latest/api/config.html#alembic.config.Config.set_main_option), which explicitly recommends escaping % symbols in this way.